### PR TITLE
Decompiled ovl_En_Ma2

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by Roman971** [comment]
I'll wait for #83 to be merged before reviewing this, since it directly depends on it.

File: README-OSX.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Developing on OSX
```
Comment (Roman971) [inline README-OSX.md:1]: I believe this is already in the repo under a slightly different name?

File: src/overlays/actors/ovl_En_Ma2/z_en_ma2.c [hunk 2]
```diff
@@ -19,39 +30,382 @@ const ActorInit En_Ma2_InitVars = {
...
+    }
+    if ((!(gSaveContext.eventChkInf[1] & 0x100)) && (globalCtx->sceneNum == 0x36) && (gSaveContext.nightFlag == 0) &&
+        (this->actor.shape.rot.z == 5)) {
+        return 1;
+    }
+    if ((!(gSaveContext.eventChkInf[1] & 0x100)) && (globalCtx->sceneNum == 0x63) && (gSaveContext.nightFlag == 1) &&
+        (this->actor.shape.rot.z == 6)) {
+        return 2;
+    }
+    if ((!(gSaveContext.eventChkInf[1] & 0x100)) || (globalCtx->sceneNum != 0x63)) {
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Ma2/z_en_ma2.c:175]: Scene enum values should be used here.

File: README-OSX.md (hunk 1 above)
Comment (ethteck) [reply README-OSX.md:1]: Yeah, thanks. Forgot I renamed it and this is left over.

File: src/overlays/actors/ovl_En_Ma2/z_en_ma2.c (hunk 2 above)
Comment (ethteck) [reply src/overlays/actors/ovl_En_Ma2/z_en_ma2.c:175]: like SCENE_ZERO, SCENE_ONE, ???

Comment (Roman971) [reply src/overlays/actors/ovl_En_Ma2/z_en_ma2.c:175]: `globalCtx->sceneNum == SCENE_...`

Comment (ethteck) [reply src/overlays/actors/ovl_En_Ma2/z_en_ma2.c:175]: ohhhh psh, those scenes. oops. I thought you meant to create a new scene enum for this actor and use it as the return values. I'll do this in a bit

**Comment by ethteck** [comment]
Should be good to go.

**Pull request merged**